app/jobs/search_keyword_job.rb

```diff
@@ -5,22 +5,38 @@ class SearchKeywordJob < ApplicationJob
 
   def perform(search_result_id)
     search_result = SearchResult.find(search_result_id)
-    html_code = Search::SearchService.new(
-      keyword: search_result.keyword,
-      search_engine: search_result.search_engine
-    ).search
+    html_code = search(search_result.keyword, search_result.search_engine)
 
-    return mark_as_failed(search_result) unless html_code
+    result = parse(search_result.search_engine, html_code)
```

Comment:
Could we rename this to `parsing_result` / `parse_result`, for more clarity? 🙇 

Same goes for the parameter inside `set_parsing_result` 🙏